tgui/packages/tgui/interfaces/Changelog.jsx

```diff
@@ -273,29 +273,27 @@ export class Changelog extends Component {
                   <Box ml={3}>
                     <Table>
                       {changes.map((change) => {
-                        const changeType = Object.keys(change)[0];
+                        const changeKey = Object.keys(change)[0];
+                        const changeType =
+                          changeTypes[changeKey] || changeTypes['unknown'];
```

Comment:
The slightly weird formatting here was suggested by Prettier to avoid going over 80 characters.